Invalid name error when npm install

Hi Ben,

I really like this bookmarklet, this is what I've been looking for!! Thank you for making it an OSS project 😄  I just wanted to try building it locally but I encountered an error below:

```
$ npm i
npm WARN Invalid name: "Copy issue link Bookmarklet"
npm WARN copy-issue-link-bookmarklet No description
npm WARN copy-issue-link-bookmarklet No repository field.
npm WARN copy-issue-link-bookmarklet No README data
npm WARN copy-issue-link-bookmarklet No license field.
```

The log level is Warning, but actually it didn't install dependencies:

```
$ npm list
/Users/lowply/.ghq/github.com/benbalter/copy-issue-link-bookmarklet
└── (empty)

npm ERR! error in /Users/lowply/.ghq/github.com/benbalter/copy-issue-link-bookmarklet: Invalid name: "Copy issue link Bookmarklet"

$ ls -l node_modules/
total 0
```

It looks like there's some rules for `name` field. Please see [package.json](https://docs.npmjs.com/files/package.json)

So I tried

```
  "name": "copy-issue-link-bookmarklet",
```

And it worked. Hope it helps 😄

--- a/package.json
+++ b/package.json
@@ -1,5 +1,5 @@
 {
-  "name": "Copy issue link Bookmarklet",
+  "name": "copy-issue-link-bookmarklet",
   "version": "0.0.1",
   "description": "Copies a markdown link to the issue your viewing",
   "main": "dist/bookmark.js",

